securityscanutils/securityscan.go

```diff
@@ -96,18 +109,24 @@ func (s *SecurityScanner) GenerateSecurityScans(ctx context.Context) error {
 	}()
 	for _, repo := range s.Repos {
 		opts := repo.Opts
-		allReleases, err := githubutils.GetAllRepoReleases(ctx, client, repo.Owner, repo.Repo)
+		allReleases, err := githubutils.GetAllRepoReleases(ctx, s.githubClient, repo.Owner, repo.Repo)
 		if err != nil {
 			return eris.Wrapf(err, "unable to fetch all github releases for github.com/%s/%s", repo.Owner, repo.Repo)
 		}
 		// Filter releases by version constraint provided
 		filteredReleases := githubutils.FilterReleases(allReleases, opts.VersionConstraint)
 		githubutils.SortReleasesBySemver(filteredReleases)
+		if repo.Opts.CreateGithubIssuePerImageVulnerability {
+			repo.allGithubIssues, err = githubutils.GetAllIssues(ctx, s.githubClient, repo.Owner, repo.Repo, &github.IssueListByRepoOptions{})
```

Comment:
(1) Can we use these options to specify a label or something that would reduce the number of results returned?  IIUC, we're fetching all the issues just so we can check if an issue has already been created for this vulnerability? If we're creating a new vulnerability issue we could apply the label during creation, then use it to reduce the size of this query.

(2) Does `githubutils.GetAllIssues` return only open issues, or is that another option we could potentially specify here?

Reply:
1) Good idea! 👍 

2) By default it returns only open issues :)